soh/soh/Enhancements/randomizer/3drando/location_access/locacc_castle_town.cpp

```diff
@@ -135,14 +135,16 @@ void AreaTable_Init_CastleTown() {
                   Entrance(CASTLE_GROUNDS, {[]{return true;}}),
   });
 
-  areaTable[GANONS_CASTLE_GROUNDS] = Area("Ganon's Castle Grounds", "Castle Grounds", OUTSIDE_GANONS_CASTLE, NO_DAY_NIGHT_CYCLE, {}, {
+  areaTable[GANONS_CASTLE_GROUNDS] = Area("Ganon's Castle Grounds", "Castle Grounds", OUTSIDE_GANONS_CASTLE, NO_DAY_NIGHT_CYCLE, {
+                  EventAccess(&BuiltRainbowBridge, {[]{return CanBuildRainbowBridge;}}),
+                }, {
                   //Locations                                   //the terrain was lowered such that you can't get this GS with a simple sword slash
                   LocationAccess(OGC_GS, {[]{return HasExplosives || (IsAdult && (LogicOutsideGanonsGS || Bow || HookshotOrBoomerang || CanUse(DINS_FIRE)));}}),
```

Comment:
With the other changes we now know only adult will have access at this point.
Based on the message here, 3ds has implemented a logic trick for a precise jumpslash to get this GS, as the level geometry is different than n64.
Comparing against N64 glitchless logic, they have this simply as a `return true` since you can just walk up, use a simple sword slash to kill the skull, and can jump/hang off the ledge to get the token.

Thoughts on us simplifying and matching n64 logic in this case while we are already here?

Reply:
I already changed this when going through for the trick logic PR but would be nice to have it done with this honestly. Every type of slash can reach the gs from my testing so would make sense on our side to have it return true

Reply:
I can change it to `return true` for now since we can always assume Adult Link has a sword, but when Master Sword Shuffle becomes a thing we would need to revisit that. We do eventually want Master Sword Shuffle right?

Reply:
Used `CanJumpslash || CanUseProjectile || (CanShield && CanUse(MEGATON_HAMMER) || CanUse(DINS_FIRE)`, which should cover all the ways Adult Link can attack the Skulltula (only Megaton Hammer Crouch Stabs can reach, which require having a shield.